backend/data_validation/base_data_val.py

```diff
@@ -0,0 +1,228 @@
+import collections
+import json
+import statistics
+
+import falcon_alliance
+import numpy as np
+import yaml
+from utils import *
```

Comment:
Generally it's good practice to import only the functionality you need from a library (unless it's A LOT) of things. It also makes it easier to identify exactly which library a certain function/class is coming from.

See this for example: https://stackoverflow.com/a/2386740